backend/fms_core/resources/experiment_run.py

```diff
@@ -0,0 +1,279 @@
+from django.core.exceptions import ValidationError
+from django.contrib.contenttypes.models import ContentType
+import reversion
+from import_export.fields import Field
+from import_export.widgets import ForeignKeyWidget, DateWidget
+from datetime import datetime
+from itertools import accumulate, islice
+import json
+
+from ..models import (
+    ExperimentRun,
+    Container,
+    Instrument,
+    ExperimentType,
+    Protocol,
+    Process,
+    ProcessMeasurement,
+    PropertyType,
+    PropertyValue,
+    Sample,
+    SampleLineage
+)
+from ._generic import GenericResource
+
+from ._utils import skip_rows, add_columns_to_preview, wipe_import_row_result
+from ..utils import (
+    blank_str_to_none,
+    float_to_decimal,
+    get_normalized_str,
+)
+
+from ..experiments import PROTOCOLS_BY_EXPERIMENT_TYPE_NAME
+
+__all__ = ["ExperimentRunResource"]
+
+
+process_content_type = ContentType.objects.get(app_label="fms_core", model="process")
+
+
+class ExperimentRunResource(GenericResource):
+    experiment_id = Field(column_name='Experiment ID')
+    container_barcode = Field(column_name='Experiment Container Barcode',
+                              widget=ForeignKeyWidget(Container, field='barcode'))
+    container_kind = Field(column_name='Experiment Container Kind')
+    instrument_name = Field(column_name='Instrument Name',
+                              widget=ForeignKeyWidget(Instrument, field='name'))
+    start_date = Field(attribute='start_date', column_name='Experiment Start Date', widget=DateWidget())
+
+
+    ERROR_CUTOFF = 500
```

Comment:
Is this for testing ? What is the reason for changing the cutoff here ? Add a comment to the code with your reason. it is not obvious to me.

Reply:
If there is a problem with only one row, it would stop being processed very early (there is a lot of data in only one row). 
This is why I wanted to increase the cutoff number. 500 is abritrary we can choose another value higher than 20, and I will add a comment if that's ok with you.

Reply:
yeah we'll need to do some error cleanup here too ... but yeah for now a comment will do just fine and we'll tweak as we cleanup the error messages generated...